Pocket/pocket-ios
# Fix collection title and image url broken under certain conditions

## Goal
* Include short description of what this PR accomplishes

## Test Steps
* Save the following item on web
https://getpocket.com/collections/why-habits-matter-and-how-to-rewire-yours-for-good?utm_source=pocket&utm_medium=email&utm_campaign=pockethits&cta=1&src=ph
* make sure title and image appear correctly both in the recent saves and in the saves lists
